src/styles/cards/cards_blocks/card.scss

```
@@ -0,0 +1,91 @@
+@use "./cards_utils/__variables" as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
```

In the future - Do not add extra files to PR
To add the file to a commit use git add src/.
```suggestion
  background-color: $c-white;
```